docs/example-intake-events.asciidoc

```diff
@@ -1,4 +1,4 @@
-[[example-intakev2-events]]
+[[example-intake-events]]
```

Comment:
Whenever we change block IDs, we should add the old IDs to the `redirects.asciidoc` file. This file is hidden and ensures that any external links to these pages don't break by creating a redirect landing page for the old links.

File: `redirects.asciidoc`
Text to add:
```
[role="exclude",id="example-intakev2-events"]
=== Example Request Body

This page has moved. Please see <<example-intake-events>>.
```

Reply:
I added all removed IDs to the redirect.

Reply:
Thank you! Just tested them all and everything looks good